recipe/meta.yaml

```
@@ -92,6 +68,25 @@ test:
     - shap.utils
     - shap.actions
     - shap.models
+  commands:
+    - pip check
+    # OMP: Error #13: Assertion failure at kmp_dispatch.cpp(1298).
+    # https://github.com/llvm/llvm-project/issues/54422
+    - export OMP_NUM_THREADS=1  # [osx]
+    - pytest -sv tests {{ ignore_tests }} --ignore=tests/benchmark
+  requires:
+    - pip
+    - pytest
+    - matplotlib-base
+    - ipython
+    - xgboost
+    - pytorch  # [not win]
+    - transformers  # [not win]
```

```suggestion
    - pytorch
    - transformers
```
Both are available on windows.